packages/create-nuxt-app/lib/prompts.js

```diff
@@ -107,6 +107,11 @@ module.exports = [
     ],
     default: 'server'
   },
+  {
+    name: 'srcDir',
+    message: 'Src directory:',
```

Comment:
Any suggestion for better question message?

Reply:
What about something like:
```suggestion
    message: 'Source directory, relative to root:',
```

Reply:
Thank you for suggestion!
I applied your code and modified same kind of wording as well: b1be40b4988e3bef23b56d8c0f8a73a67d7e214f